### PR TITLE
ENH: Communication practices

### PR DESCRIPTION
**Pull request by choldgraf**
This documents our team's external communication practices. It tries to be clear that our goal is not to gatekeep communication, but to share information across the team and give others a chance to participate. It also generalizes our `talking.rst` (which was focused on live talks) to instead be an `external-communication.rst` page that also covers talks and blog posts.

### CODE REVIEW

**Review by sgibson91 — APPROVED** [review]

**Comment by sgibson91** [comment]
This may clash with Erik's PR regarding rst2myst, but I'm sure you're already aware :)

**Review by betatim — APPROVED** [review]

**Comment by betatim** [comment]
I like it, especially the "stick to past and present". It is a great formulation for an idea for which I only had long winded and verbose explanations.

File: docs/resources/shared-infrastructure.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Some of these are useful for our development workflows, while others are used as
 We have [a GitHub organization](https://github.com/jupyterhub/) for hosting all of our code repositories.
 This organization is where we do most of the code-related work for the project, and where we have discussions and coordination.
 
+(resources:shared-drive)=
+
 ## Google Drive
 
 We have a [**JupyterHub Team Google Drive**](https://drive.google.com/drive/u/1/folders/0B8VZ4vaOYWZ3X29KTzZSemlNSG8) to store documents, presentations, images, etc that are useful to the team.
```
Comment (manics) [inline docs/resources/shared-infrastructure.md:15]:
```suggestion
We have a [**JupyterHub Team Google Drive**](https://drive.google.com/drive/u/0/folders/0B8VZ4vaOYWZ3X29KTzZSemlNSG8) to store documents, presentations, images, etc that are useful to the team.
```

The `/u/1/` in the Google drive URL means access wll be requested from the users's second Google account if they're logged in to multiple accounts. This should probably be `/u/0/`?

Comment (choldgraf) [reply docs/resources/shared-infrastructure.md:15]: oops good point - I replaced it with the `shared link` which is user-agnostic

Comment (choldgraf) [inline docs/resources/shared-infrastructure.md:15]:
```suggestion
We have a [**JupyterHub Team Google Drive**](https://drive.google.com/drive/folders/0B8VZ4vaOYWZ3X29KTzZSemlNSG8?resourcekey=0-7kPY41v8FRajEmoW0Ojimg&usp=sharing) to store documents, presentations, images, etc that are useful to the team.
```

**Comment by choldgraf** [comment]
re-ran rst2myst on this so it should now be ready to go again! If I don't hear any objections, I'll take @betatim's approval as applying here as well and merge tomorrow AM (my time)

**Review by consideRatio — APPROVED** [review]
Nice I really appreciate how there is now a section about blog post communication!

This would absolutely reduce the barrier for me to contributing with such communication if it could become suitable, something I've never done before.

**Comment by consideRatio** [comment]
Related to the discussion in #594 about merging, this is a PR that I figure merits more approval than the typical PR - so merging on the first approval would been a bit fast. But with five reviews and resolved discussion, I think its good time to go for a merge and that its more helpful if I do it rather than await @choldgraf to do it as the author.

**Pull request merged**